doc/ug/getting_started_dv.md

```diff
@@ -88,3 +88,23 @@ Please refer to the [checklist]({{< relref "doc/project/checklist" >}}) to under
 
 The [UART DV](https://github.com/lowRISC/opentitan/tree/master/hw/ip/uart/dv) area can be used as a canonical example for making progress.
 If it is not clear on how to proceed, feel free to file an issue requesting assistance.
+
+## Reproduce DV failure in CI(private)
+
+Follow these steps to reproduce CI(private) failure
+
+1. Make sure the version of VCS is the same as the [one](https://github.com/lowRISC/opentitan-private-ci/blob/master/jobs.yml#L5) running in CI
+
+2. CI runs against an auto-generated merge commit, which effectively is the PR merge into master.
+This "merge" branch gets updated automatically by GitHub whenever you push to the PR branch, or when you close/re-open the PR.
+Get the extra database by doing the following (Assuming "upstream" is the name of your lowrisc/opentian repository)
```

Comment:
```suggestion
Retrieve this database by running (assuming "upstream" is the name of your lowrisc/opentian repository):
```

Reply:
Can we avoid the term "database"? It always confuses me when I read it -- a directory full of files is just not a database in the general use of the term.